metriq_gym/dispatch_clops.py

```diff
@@ -18,36 +18,35 @@
 def main():
     args = parse_arguments()
 
-    if not args.token:
-        raise ValueError("CLOPS benchmark requires an IBMQ token!")
-
-    QiskitRuntimeService.save_account(channel="ibm_quantum", token=args.token, set_as_default=True, overwrite=True)
+    if args.token:
+        QiskitRuntimeService.save_account(channel="ibm_quantum", token=args.token, set_as_default=True, overwrite=True)
 
     logging.info(f"Dispatching CLOPS job with n={args.num_qubits}, shots={args.shots}, trials={args.trials}, backend={args.backend}, confidence_level={args.confidence_level}, jobs_file={args.jobs_file}")
 
     clops = clops_benchmark(
-        QiskitRuntimeService(),
-        args.backend,
-        width = args.num_qubits,
-        layers = args.num_qubits,
-        shots = args.shots
+        service=QiskitRuntimeService(),
+        backend_name=args.backend,
+        width=args.num_qubits,
+        layers=args.num_qubits,
+        shots=args.shots
     )
     
-    result = BenchJobResult(
+    partial_result = BenchJobResult(
         id = clops.job.job_id(),
+        backend=args.backend,
         provider = BenchProvider.IBMQ,
         job_type = BenchJobType.CLOPS,
-        qubits = clops.job_attributes.width,
-        shots = clops.job_attributes.shots,
-        depth = clops.job_attributes.layers,
+        qubits = clops.job_attributes["width"],
```

Comment:
Instead of using two different names here (qubits and width) should we just refer to this unanimously as "qubits"? It's somewhat confusing to have two names referring to the same thing.